### PR TITLE
Metrics: F1Beta, MCC

### PR DESCRIPTION
**Pull request by jegorus**
RT-6. Added f1beta, mcc and tests

### CODE REVIEW

File: rectools/metrics/classification.py [hunk 1]
```diff
@@ -424,3 +425,75 @@ def make_confusions(reco: pd.DataFrame, interactions: pd.DataFrame, k: int) -> p
     merged = merge_reco(reco, interactions)
     confusion_df = calc_confusions(merged, k)
     return confusion_df
+
+
+@attr.s
+class F1Beta(SimpleClassificationMetric):
+    """
+    Calculates metric base on precision@k and recall@k
```
Comment (feldlime) [inline rectools/metrics/classification.py:433]:
```suggestion
    Fbeta score for k first recommendations.
```

File: rectools/metrics/classification.py [hunk 2]
```diff
@@ -424,3 +425,75 @@ def make_confusions(reco: pd.DataFrame, interactions: pd.DataFrame, k: int) -> p
...
+        - ``fp`` - number of non-relevant recommendations among the first `k` items of recommendation list;
+        - ``fn`` - number of items the user has interacted with but that weren't recommended (in top-`k`).
+
+    Parameters
+    ----------
+    k : int
+        Number of items in top of recommendations list that will be used to calculate metric.
+    """
+
+    def _calc_per_user_from_confusion_df(self, confusion_df: pd.DataFrame, catalog: Catalog) -> pd.Series:
```
Comment (feldlime) [inline rectools/metrics/classification.py:490]: why do you need `catalog` here? And why don't you use `SimpleClassificationMetric` as the base class?

**Review by feldlime — CHANGES_REQUESTED** [review]

File: rectools/metrics/classification.py (hunk 2 above)
Comment (jegorus) [reply rectools/metrics/classification.py:490]: I need it to use `TN`, which is calculated in `ClassificationMetric` using `catalog`:
`confusion_df[TN] = len(catalog) - self.k - confusion_df[FN]` (from function `calc_per_user_from_confusion_df`)

File: rectools/metrics/classification.py (hunk 1 above)
Comment (jegorus) [reply rectools/metrics/classification.py]: changed

File: rectools/metrics/classification.py (hunk 2 above)
Comment (feldlime) [reply rectools/metrics/classification.py:490]: ok

**Comment by sharthZ23** [comment]
Approve

**Pull request merged**